### PR TITLE
AAP-29069 Create Operator section of Getting Started guide

### PR DESCRIPTION
**Pull request by ariordan-redhat**
AAP-29069
Affects `/titles/getting-started`

### CODE REVIEW

File: downstream/modules/platform/proc-gs-downloading-content.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[id="gs-downloading-content_{context}"]
+
+= Downloading content 
+
+After collections are finalized, you can import them to a location where they can be distributed to others across your organization.
+
+.Procedure
+
+. Log in to {PlatformName}.
+. From the navigation panel, select *Full Experience*.
```
Comment (hherbly) [inline downstream/modules/platform/proc-gs-downloading-content.adoc:10]: The full experience/tech preview thing is gone--I would remove this step. It's just logging into AAP and then navigating to Automation Content > wherever you're going.

File: downstream/modules/platform/proc-gs-downloading-content.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[id="gs-downloading-content_{context}"]
+
+= Downloading content 
+
+After collections are finalized, you can import them to a location where they can be distributed to others across your organization.
+
+.Procedure
+
+. Log in to {PlatformName}.
+. From the navigation panel, select {MenuACCollections}.
```
Comment (ariordan-redhat) [inline downstream/modules/platform/proc-gs-downloading-content.adoc:10]: @hherbly Is this correct now?

Comment (hherbly) [reply downstream/modules/platform/proc-gs-downloading-content.adoc:10]: Yes! ty ty

**Review by hherbly — APPROVED** [review]
LGTM

**Pull request merged**